docs/components/DrawerDocs.js

```diff
@@ -212,9 +212,9 @@ const DrawerDocs = React.createClass({
     }
 
     HeaderMenu.propTypes = {
-      handleButtonClick: React.PropTypes.func,
-      handleScrimClick: React.PropTypes.func,
-      showSimpleSelectMenu: React.PropTypes.bool
+      handleButtonClick: PropTypes.func,
+      handleScrimClick: PropTypes.func,
+      showSimpleSelectMenu: PropTypes.bool
```

Comment:
This is just markdown for the docs, but i feel like it's probably better to do this in our markdown anyway? maybe out of the scope of this PR though... what do you think